### ISSUE TITLE
Forbidding use of boilerplate tools like Create React App is irrelevant to testing someone's problem solving abilities

### ISSUE THREAD

**Issue by blenderer** [Superformula/frontend-test#7]
I appreciate the explanation in README, but I disagree with forbidding the use of boilerplating like Create React App. After reading the following, I feel like the explanation here is contradictory. Specifically, asking to show how they solve a problem, then restricting them to solve a problem a certain way.

> Use any libraries that you would normally use if this were a real production App. Please note: we're interested in your code & the way you solve the problem, not how well you can use a particular library or feature.
> 
> We're interested in your method and how you approach the problem just as much as we're interested in the end result.

I believe that a large part of problem solving is: "Using the right tool for the job". There are very few cases in which anyone should need to bootstrap a React app without something like Create React App.

- It's free
- It's tested by hundreds of thousands of people
- It's maintained by the brightest individuals in the React community
- **The alternative to using it is copy/pasting bundler/build configuration code from webpack's documentation site**

 If a client came to me and told me to build a machine from with just my hands, a wood plank and some duct tape- I would tell them that it is unfeasible for them as well as me.

**Comment by wprater** [comment]
Thanks for the note.  But we do not use nor rely on tools like CRA for a production product, since you'll find you grow out of it.  We still stand by this I do not wish to see copy and paste code from documentation sites nor the use of something like CRA.  Please think about what is needed for this PR to solve the problem and present them clearly in your commits.

**Comment by blenderer** [comment]
I think the outgrowing problem used to be the case in the early days of CRA for sure.

This is exactly what my team thought and we eventually ejected our initial React apps.

However, once we scaled to more than 2 apps, managing builds, bundles and deployments became a nightmare. Newly installed dependancies would still be using our locked down, old webpack/babel versions as we could not update them automatically. Our sourcemaps we're broken in test environments because the old webpack build had not fixed a bug that was introduced in Chrome later on.

We eventually forced all of our repos to use `react-scripts` and it created a lot of consistency. With the click of a button, we had the ability to instantly start deploying to our CDN without spending hours configuring a webpack to understand the code that we all write every day.

Our philosophy was that we'd rather spend time building things fast and not having to worry about "reinventing the wheel" every time. 

I understand the need to avoid vendor lock-in and I'm on board, so I'll close this issue.
